TCAT/WhatsNewHeaderView.swift

```diff
@@ -0,0 +1,155 @@
+//
+//  WhatsNewPopupView.swift
+//  TCAT
+//
+//  Created by Omar Rasheed on 9/26/18.
+//  Copyright © 2018 cuappdev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+protocol WhatsNewDelegate {
+    func okButtonPressed()
+    func cardPressed()
+}
+
+class WhatsNewHeaderView: UIView {
+
+    var whatsNewDelegate: WhatsNewDelegate?
+    var updateTitle: UILabel!
+    var dismissButton: UIButton!
+    var updateDescription: UILabel!
+    var whatsNewHeader: UILabel!
+    var backgroundView: UIView!
+    
+    override init(frame: CGRect) {
```

Comment:
We will want to be able to reuse this for future features, so I would recommend abstracting out the text into this class' initializer or even create a separate class just for the metadata. Up to you on the implementation, but all the variable text should be in one place and easily changeable instead of interspersed throughout the setup functions.